<?xml version="1.0"?>

<launch>
  <!-- display node for the detections -->
  <!--rosparam command="load" ns="cob_people_detection/people_detection_display" file="$(find cob_people_detection)/ros/launch/people_detection_display_params.yaml"/-->
  <node pkg="cob_people_detection" ns="cob_people_detection/people_detection_display" type="people_detection_display_node" name="people_detection_display" output="screen">
    <remap from="face_position_array" to="/cob_people_detection/detection_tracker/face_position_array"/>
    <remap from="face_detections" to="/cob_people_detection/face_detector/face_positions"/>
    <remap from="colorimage" to="/camera/rgb/image_color"/>
  </node>

  <!-- Image viewer display -->
  <!--node name="people_detection_viewer" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/cob_people_detection/people_detection_display/face_position_image"/>
  </node-->

</launch>
